### PR TITLE
Fix/ Disabled services info in statusCommand

### LINKED ISSUE #203: Status command shows provider and indexer info even if they are not available/enabled
This can be somehow misleading
Example response (without DB and therefore no indexer nor provider):
```
{
  "id": "16Uiu2HAkuYfgjXoGcSSLSpRPD6XtUgV71t5RqmTmcqdbmrWY9MJo",
  "publicKey": "080212210201cabbabef1cc85218fa2d5bbadfb3425dfc091b311a33e6d9be26f6dcb94668",
  "address": "0x8F292046bb73595A978F4e7A131b4EBd03A15e8a",
  "version": "1.0.0",
  "http": true,
  "p2p": true,
  "provider": [
    {
      "chainId": "11155420",
      "network": "optimism-sepolia"
    }
  ],
  "indexer": [
    {
      "chainId": "11155420",
      "network": "optimism-sepolia",
      "block": "0"
    }
  ],
  "supportedStorage": {
    "url": true,
    "arwave": true,
    "ipfs": true
  },
  "uptime": 145.47725728,
  "platform": {
    "cpus": 8,
    "freemem": 8240238592,
    "totalmem": 16442556416,
    "loadavg": [
      0.97,
      1.07,
      1.02
    ],
    "arch": "x64",
    "machine": "x86_64",
    "platform": "linux",
    "release": "6.2.0-39-generic",
    "osType": "Linux",
    "osVersion": "#40~22.04.1-Ubuntu SMP PREEMPT_DYNAMIC Thu Nov 16 10:53:04 UTC 2",
    "node": "v18.19.0"
  }
}
```

### PR DESCRIPTION
**Pull request by bogdanfazakas**
Fixes #203  .

Changes proposed in this PR:

- 
- 
-

### CODE REVIEW

**Review by paulo-ocean — APPROVED** [review]
lgtm

**Review by mariacarmina — APPROVED** [review]
lgtm

File: src/components/core/utils/statusHandler.ts [hunk 1]
```diff
@@ -72,17 +72,21 @@ export async function status(
...
-      status.provider.push(provider)
-      const indexer: OceanNodeIndexer = {
-        chainId: key,
-        network: supportedNetwork.network,
-        block: '0'
+      if (config.hasIndexer) {
+        const indexer: OceanNodeIndexer = {
+          chainId: key,
+          network: supportedNetwork.network,
+          block: '0'
```
Comment (alexcos20) [inline src/components/core/utils/statusHandler.ts:86]: we should get the actual block no, but we can do it in another issue

Comment (bogdanfazakas) [reply src/components/core/utils/statusHandler.ts:86]: should be pretty straight fwd, will add it here

**Pull request merged**